manifests/vhost.pp

```diff
@@ -1717,6 +1717,12 @@
 #   client request exceeds that limit, the server will return an error response
 #   instead of servicing the request.
 #
+# @param $use_servername_for_filenames
+#   When set to true, default log / config file names will be derived from the sanitized
+#   value of the $servername parameter.
+#   When set to false (default), the existing behaviour of using the $name parameter
+#   will remain.
+
```

Comment:
This broke puppet-strings: https://github.com/puppetlabs/puppetlabs-apache/pull/2165.

Yet another reason why I hate this change and would still be strongly in favor of reverting it.